index.html

```diff
@@ -6,13 +6,14 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Medini Website Design</title>
     <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
 </head>
 <body>
     <header class="container">
         <a><h3 class="main_heading">My Logo</h3></a>
         <nav>
-            <img class="menubutton" src='./images/Union.svg' alt="menubutton">
+            <img class="menubutton" id="menuicon" src='./images/Union.svg' alt="menubutton">
         </nav>
         <ul class="header_list_items">
             <li>Portfolio</li>
```

Comment:
- [ ] The _menu items_ on your mobile menu **do not send the user to the targeted** part of the page. Kindly ensure that when a user clicks (or taps) any of the mobile menu options, a correct part of the page is displayed.

Reply:
This issue is well resolved! ✔️